compiler/quilt/tools/command.py

```diff
@@ -230,7 +230,7 @@ def _handle_response(resp, **kwargs):
         except ValueError:
             raise CommandException("Unexpected failure: error %s" % resp.status_code)
 
-def _create_auth(team):
+def _create_auth(team=None):
```

Comment:
I think we should be strict about team parameters. The only place where it should be optional is public APIs that would break otherwise.